src/hotspot/share/opto/superwordVTransformBuilder.cpp

```diff
@@ -139,51 +142,72 @@ void SuperWordVTransformBuilder::build_inputs_for_scalar_vtnodes(VectorSet& vtn_
       init_req_with_scalar(n, vtn, 0);
       continue;
     } else {
-      set_all_req_with_scalars(n, vtn);
+      init_all_req_with_scalars(n, vtn);
     }
   }
 }
 
 // Create a vtnode for each pack. No in/out edges set yet.
 VTransformVectorNode* SuperWordVTransformBuilder::make_vector_vtnode_for_pack(const Node_List* pack) const {
-  uint pack_size = pack->size();
   Node* p0 = pack->at(0);
-  int opc = p0->Opcode();
-  VTransformVectorNode* vtn = nullptr;
+  const VTransformVectorNodePrototype prototype = VTransformVectorNodePrototype::make_from_pack(pack, _vloop_analyzer);
+  const int  sopc = prototype.scalar_opcode();
```

Comment:
```suggestion
  const int sopc = prototype.scalar_opcode();
```
Nit: whitespace
Or were you trying to align with the line below? Personally, I find this a bit too much, but up to you.

Reply:
I'm also refactoring away some of the assignments, and put the value directly at the use-site.